src/components/overview/ImageGallery/ImageGallery.jsx

```diff
@@ -81,7 +83,7 @@ const ImageGallery = ({state, updateState}) => {
       X = X / mWidth * 100;
       Y = Y / mHeight * 100;
 
-      mainImageElement.style.transform = 'translate(-'+X+'%, -'+Y+'%) scale(2.0)';
+      mainImageElement.style.transform = 'translate(-' + X + '%, -' + Y + '%) scale(2.0)';
```

Comment:
Might use interpolation (`text${JS value}`) here instead?

Reply:
I tried that with using as the below but somehow it did not work before.. so I kept this way.
mainImageElement.style.transform = `'translate(-${X}%, -${Y}%) scale(2.0)'`;

But I found out that it did not work because I did not delete the single quotation sign.
thanks!